package.json

```
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www"
+    "start": "node ./bin/www",
+		"test": "jest --watch"
```

```suggestion
		"test": "jest --coverage --watch"
```
This will give us the coverage percentage.